space_packet_parser/xtce/definitions.py

```diff
@@ -455,17 +503,8 @@ def packet_generator(
             point for parsing. Default is taken from the parent XtcePacketDefinition object.
         ccsds_headers_only : bool
             Default False. If True, only parses the packet headers (does not use the provided packet definition).
-            ``space_packet_parser.packets.ccsds_packet_generator`` can be used directly to parse only the CCSDS headers
+            ``space_packet_parser.ccsds.ccsds_packet_generator`` can be used directly to parse only the CCSDS headers
```

Comment:
Should the `ccsds_headers_only` kwarg be deprecated since `packet_generator` is supposed to be generic?